mautic/mautic
# Fix form submit messages length

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" 
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no




#### Description:
Form validation messages were generating "errors 500" when longer than 191 char. This PR fixes it.



#### Steps to test this 
1. Run DB migrations
2. Create a form add a long text (over 300 chars) in the form validation message input
3. Submit it
4. See that there is no errors
